mlflow/entities/span_event.py

```diff
@@ -8,6 +8,8 @@
 from opentelemetry.util.types import AttributeValue
 
 from mlflow.entities._mlflow_object import _MlflowObject
+from mlflow.entities.assessment import set_pb_value
```

Comment:
should this be from the utils file?

```suggestion
from mlflow.utils.proto_json_utils import set_pb_value
```